src/statement.js

```diff
@@ -58,11 +58,12 @@ class Statement {
    * @param  mixed  value The value to set.
    * @return mixed         The setted value.
    */
-  data(name, value) {
-    if (!arguments.length) {
+  data(...args) {
+    if (!args.length) {
       throw new Error('Missing name argument.');
     }
-    if (arguments.length === 2) {
+    const [name, value] = args
```

Comment:
this could cause an out of bounds lookup if only 1 argument is passed. That can be a performance hit. Maybe move this down into the args.length conditional body?